src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.BlazorWasmCompanionExtension/src/acquireDotnetInstall.ts

```diff
@@ -0,0 +1,29 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+
+interface IDotnetAcquireResult {
+    dotnetPath: string;
+}
+
+export async function acquireDotnetInstall(outputChannel: vscode.OutputChannel): Promise<string> {
+    const version = "5.0.0";
```

Comment:
You only need to provide the major.minor version of the runtime you want, the extension will determine what the latest patch is and acquire that.